WooCommerce/Resources/AppStoreStrings.pot

```diff
@@ -57,11 +57,13 @@ msgctxt "app_store_promo_text"
 msgid "Run your store from anywhere"
 msgstr ""
 
-msgctxt "v8.2-whats-new"
+msgctxt "v8.3-whats-new"
```

Comment:
@mokagio Another thing I never realized until now and which might be worth noting for your work on the `.po` generation code for our project: notice how for WCiOS we use `vX.Y-whats-new` as the key name as opposed to `release_note_XY` like for the other apps… which has the side effect that this is treated just like any other regular entry in the `.po` and not a special release notes block, and especially it does not keep both the release notes for NNN-1 and NNN like the action does for all the other apps.

Not sure if ultimately we do want to keep the release notes for NNN-1 (in case submission of NNN-1 is delayed because of rejection and isn't finalized when code freeze and release notes for NNN land, think recent JPiOS 18.x rejection?) or not in general for all apps — we might have to discuss that during your refactoring of po generation action — but in any case it would be nice to use that occasion to unify all apps in that regard.